week2/homework/src/Methods.js

```diff
@@ -0,0 +1,131 @@
+'use strict';
+
+const util = require('util');
+const fs = require('fs');
+
+const readFilePromise = util.promisify(fs.readFile);
+const writeFilePromise = util.promisify(fs.writeFile);
+const appendFilePromise = util.promisify(fs.appendFile);
+
+class Methods {
+  constructor(filePath, args) {
+    this.args = args;
+    this.filePath = filePath;
+  }
+
+  async list() {
+    const todos = await this.getFileData(this.filePath);
+
+    if (!todos) {
+      return console.log(`There is no data in '${this.filePath}' file!`);
+    }
+
+    console.log(`------\nTodos:\n------`);
+    todos
+      .trim()
+      .split('\n')
+      .forEach((todo, index) => {
+        console.log(`${index + 1}. ${todo}`);
+      });
+  }
+
+  getFileData(filePath) {
+    return readFilePromise(filePath, 'utf8');
+  }
+
+  write(data) {
+    return writeFilePromise(this.filePath, data, 'utf8');
+  }
+
+  async add() {
+    try {
+      const todos = this.args.join('\n');
```

Comment:
As I understand the assignment, this should be one item
`this.args.join(' ')`

Reply:
I just wanted to make it to be able to add multiple items at once.